CandidateApp.ConsoleApp/Brokers/StorageBrokers/StorageBroker.Candidate.cs

```diff
@@ -0,0 +1,20 @@
+﻿// ------------------------------------------------
+// Copyright (c) MumsWhoCode. All rights reserved.
+// ------------------------------------------------
+
+using System.Collections.Generic;
+using CandidateApp.ConsoleApp.Models.Candidate;
+
+namespace CandidateApp.ConsoleApp.Brokers.StorageBrokers
+{
+    partial class StorageBroker : IStorageBroker
+    {
+        List<Candidate> Candidates = new List<Candidate>();
+        public Candidate InsertCandidate(Candidate candidate)
```

Comment:
We need a line break between single and multi-line code.

Reply:
Added an extra line